### PR TITLE
chore: adapt mysql addon for kbagent

### PR DESCRIPTION
**Pull request by xuriwuyun**

### CODE REVIEW

File: addons-cluster/mysql/values.yaml [hunk 1]
```diff
@@ -12,15 +12,15 @@ mode: standalone
 
 ## @param replicas specify cluster replicas
 ##
-replicas: 1
+replicas: 2
```
Comment (weicao) [inline addons-cluster/mysql/values.yaml:15]: why the mode is standalone, and the replicas is 2

File: addons/mysql/templates/_helpers.tpl [hunk 2]
```diff
@@ -198,24 +205,25 @@ roles:
...
   volumeMounts:
-    - mountPath: /kubeblocks
-      name: kubeblocks
+    - mountPath: /tools
+      name: tools
 - command:
     - cp
     - -r
-    - /xtrabackup-2.4
-    - /kubeblocks/xtrabackup
```
Comment (weicao) [inline addons/mysql/templates/_helpers.tpl:212]: do not need xtrabackup any longer?

File: addons/mysql/templates/cmpd-mysql57.yaml [hunk 3]
```diff
@@ -30,15 +30,26 @@ spec:
...
+        volumeMounts:
+          - mountPath: /tools
+            name: tools
+            {{- include "mysql.spec.runtime.common" . | nindent 6 }}
     containers:
       - name: mysql
         image: {{ .Values.image.registry | default "docker.io" }}/{{ .Values.image.repository }}:5.7.44
         imagePullPolicy: {{ default .Values.image.pullPolicy "IfNotPresent" }}
         command:
           - syncer
```
Comment (weicao) [inline addons/mysql/templates/cmpd-mysql57.yaml:50]: it seems syncer is also a supervisor of the mysqld, right?

File: addons/mysql/templates/cmpd-mysql80.yaml [hunk 4]
```diff
@@ -30,6 +30,17 @@ spec:
         volumeMounts:
           - mountPath: {{ .Values.dataMountPath }}
             name: data
+      - command:
+          - cp
+          - -r
+          - /xtrabackup-8.0
+          - /tools/xtrabackup
+        image: apecloud-registry.cn-zhangjiakou.cr.aliyuncs.com/apecloud/syncer:mysql
```
Comment (weicao) [inline addons/mysql/templates/cmpd-mysql80.yaml:38]: as we disscussed today, all image address should be configurable in values.yaml

**Review by nayutah — APPROVED** [review]

**Review by kizuna-lek — APPROVED** [review]

**Review by nayutah — APPROVED** [review]

**Pull request merged**